packages/editor/src/client/atoms/typographicTextArea.tsx

```diff
@@ -1,7 +1,6 @@
 import React, {
```

Comment:
There is an issue in this file now: 
![Screenshot from 2021-07-20 12-06-41](https://user-images.githubusercontent.com/69328362/126305943-5a99eafc-7922-4621-a25c-e2380fcde6c8.png)
You can reproduce it in article edit page => add teasers, choose an article and press on save.

Reply:
@bishoymelek i can't reproduce the error. Maybe i'm doing it wrong. What did you mean with "in article edit page"???

Reply:
@bishoymelek fixed it